config.toml.example

```diff
@@ -536,7 +547,7 @@ changelog-seen = 2
 # will make code compile faster at the expense of lower runtime performance.
 # If `incremental` is set to true above, the import limit will default to 10
 # instead of LLVM's default of 100.
```

Comment:
```suggestion
```

Reply:
Hmm, so the behavior here is a little subtle - if incremental isn't set, x.py doesn't actually pass a value here and LLVM decides for itself what the value should be. I don't think it's guaranteed to always be 100, so I do want to mention it somewhere in case it needs to be updated. What about this?
```suggestion
# instead of LLVM's default.
```

Reply:
Could you then use something like the following below

```toml
#thin-lto-import-instr-limit = if incremental { 10 } else { LLVM default (currently 100) }
```

Reply:
Done, thanks for the suggestion :)